setup.sh

```diff
@@ -14,9 +19,7 @@ npm install npm@10.8.1
 npm install
 npm install gulp-cli # shouldn't be necessary, but seems to be
 
-if [[ `/bin/hostname` == "*pub-submit*" ]]; then
```

Comment:
How do I know this submit specific code is no longer needed? setup.sh is not specific to dev box.

Reply:
this is not how the splash page generator is built any more. See https://github.com/eScholarship/cdl-splashgen